phpstan.neon

```diff
@@ -3,6 +3,7 @@ includes:
 parameters:
 	level: max
 	reportUnmatchedIgnoredErrors: false
+	treatPhpDocTypesAsCertain: false
```

Comment:
I made this change to address this error:

![image](https://github.com/user-attachments/assets/2b91d505-27ff-4c2c-a3c6-32745d459dab)

Seems like a good idea anyway.